.mailmap

```diff
@@ -373,4 +373,10 @@ Akshat Maheshwari <akshat14714@gmail.com>
 Poom Chiarawongse <eight1911@gmail.com> <Eight1911@users.noreply.github.com>
 Poom Chiarawongse <eight1911@gmail.com> <15707729+Eight1911@users.noreply.github.com>
 Leonid Kovalev <normalhuman@users.noreply.github.com>
+Leonid Kovalev <normalhuman@users.noreply.github.com> <leonid.traveling@gmail.com>
 Jashanpreet Singh <jashansingh.4398@gmail.com>
+Cristian Di Pietrantonio <cristiandipietrantonio@gmail.com>
+P. Sai Prasanth <psai.prasanth.min16@itbhu.ac.in>
+Cédric Travelletti <cedrictravelletti@gmail.com> Cedric <cedric@localhost.localdomain>
```

Comment:
Could you change to 
`Cédric Travelletti <cedrictravelletti@gmail.com>`
The second email was added because of wrong setting I had in the past.

Reply:
@CedricTravelletti That is exactly what `.mailmap` is for. It links commits with wrong address to the right address.